### ISSUE TITLE
Adding support for Live View interferes with non-live Delete functionality

### ISSUE THREAD

**Issue by ijt** [phoenixframework/phoenix_live_view#1339]
tldr: see https://github.com/ijt/delbug.

### Environment

* Elixir version (elixir -v):

Erlang/OTP 23 [erts-11.1.3] [source] [64-bit] [smp:6:6] [ds:6:6:10] [async-threads:1] [hipe] [dtrace]

Elixir 1.11.2 (compiled with Erlang/OTP 23)

* Phoenix version (mix deps):
  locked at 1.5.7 (phoenix) 774cd644

* Phoenix LiveView version (mix deps):
  locked at 0.15.4 (phoenix_live_view) 35d78f3c

* NodeJS version (node -v):
v15.8.0

* NPM version (npm -v):
7.5.0

* Operating system:
Mac OS X

* Browsers you attempted to reproduce this bug on (the more the merrier): 
Brave, Safari, Chrome, Firefox

* Does the problem persist after removing "assets/node_modules" and trying again? Yes/no:

The problem does persist after running
```
cd assets && rm -rf node_modules && npm i && node node_modules/webpack/bin/webpack.js --mode development
```

### Actual behavior

`git clone git@github.com:ijt/delbug.git`

  * Install dependencies with `mix deps.get`
  * Create and migrate your database with `mix ecto.setup`
  * Install Node.js dependencies with `npm install` inside the `assets` directory
  * Start Phoenix endpoint with `mix phx.server`

To reproduce the bug, visit `localhost:4004/posts` from your browser.
Click "New Post", give it a body, and click Save.
In the Show Post page, click Back.
In the Listing Post page, click Delete on the line with your new post.
In the dialog that comes up asking "Are you sure?", click OK.
If this repro has done its job, the same dialog appears again.

### Expected behavior

The dialog should disappear after clicking OK the first time and not come back. The post should be deleted and a flash should appear saying so.

**Comment by chrismccord** [comment]
Can you explain a bit about the expected vs observed behaviour? Also what phoenix HTML version are you on? Thanks!

**Comment by ijt** [comment]
The expected behavior is for the confirmation dialog to appear only once. The actual behavior is that it keeps reappearing.

Here's the phoenix HTML version.

* phoenix_html 2.14.3 (Hex package) (mix)

Thanks for looking into this so quickly!

**Comment by chrismccord** [comment]
And this goes away if you comment out the instantiation of the `LiveSocket` in app.js? I see you aren't actually rendering a LV anywhere. If merely instantiating (and/or liveSocket.connect())  is what triggers it, can you report any js console errors? Thanks!

**Comment by ijt** [comment]
I'm not seeing any js console errors.

It doesn't go away if I comment out the instantiation of the `LiveSocket` in app.js.

I did notice though that if I put in a `console.log('hi from app.js')` that runs twice when the page loads.

**Comment by chrismccord** [comment]
Your root layout renders the HTML doc/head with the app.js script, then your app.html layout renders a nested HTML doc/head with app.js, so you are loading your app.js twice and that is causing the alert to appear twice. Update your app.html layout to only include the body contents are you should be good :)

**Comment by ijt** [comment]
Great, trying that now. Many thanks.

**Comment by ijt** [comment]
It works. Yay!
